python_sdk/infrahub_sdk/config.py

```diff
@@ -134,15 +122,16 @@ def password_authentication(self) -> bool:
             return True
         return False
 
+    @property
+    def has_proxy_mounts(self) -> bool:
+        # pylint: disable=no-member
+        return self.proxy_mounts.http is not None or self.proxy_mounts.https is not None
```

Comment:
I'm a bit surprised by the pylint part here but I assume that it's just that pylint doesn't parse the pydantic fields in a correct way.

One minor comment here I think se can move the logic here to the `ProxyMountsConfig` class so that the above property instead is:

```python
    @property
    def has_proxy_mounts(self) -> bool:
        # pylint: disable=no-member
        return self.proxy_mounts.has_mounts
```

That way the ConfigBase class doesn't need to care as much about the internals of the ProxyMountsConfig class.

Reply:
Good point, moving the property to the `ProxyMountsConfig` class will probably also make pylint happy.